src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.cs

```diff
@@ -299,6 +313,56 @@ private static bool TryGetApiData(ImmutableArray<AdditionalText> additionalTexts
             return true;
         }
 
+        private static bool TryGetEditorConfigOptionForMissingFiles(AnalyzerOptions analyzerOptions, Compilation compilation, out bool optionValue)
```

Comment:
Can't you use/reuse the logic inside `AnalyzerOptionsExtensions` here?

Reply:
Nope, public API analyzer is built on top of a very old Roslyn API for back compat reasons and we cannot reference the APIs from `AnalyzerOptionsExtensions`, which need Microsoft.CodeAnalysis version 3.x or higher

https://github.com/dotnet/roslyn-analyzers/blob/ed0bbcac3280784cb701db3a44263a27337ec599/src/PublicApiAnalyzers/Core/Analyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.csproj#L10